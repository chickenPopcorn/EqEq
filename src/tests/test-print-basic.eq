FarmTownCtx = {
  acres = { 42; }
  population = { 99; }
}

FarmTownCtx:find population {
  print("%0.0f people live in this in town\n", population);
  print("The %0.2f acres of crops is just enough\n", acres);
}
