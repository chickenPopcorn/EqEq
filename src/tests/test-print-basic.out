99 people live in this in town
The 42.00 acres of crops is just enough
